database/seeds/DatabaseSeeder.php

```diff
@@ -40,6 +40,7 @@ public function devSeeds()
         $this->call(AdminUsersSeeder::class);
         $this->call(VouchersSeeder::class);
         $this->call(VoucherStatesSeeder::class);
+        $this->call(BundleSeeder::class);
```

Comment:
We need to remember to append this to the UsabilitySeeds() file too, when we build Bobby Bundles and family.